common/src/Assets/Texture.h

```diff
@@ -102,10 +113,12 @@ namespace TrenchBroom {
 
             mutable GLuint m_textureId;
             mutable BufferList m_buffers;
+
+            GameData m_gameData;
```

Comment:
Sounds out of scope for this PR.. What stuff specifically?

Reply:
The Quake 3 specific members: `m_surfaceParams`, `m_culling`, `m_blendFunc`. So far, they were the only pieces of game specific data that we were tacking onto `Texture`. Now you introduced a more general mechanism of doing this, so I'm suggesting to use this mechanism for the Quake 3 specific data as well. I think it's in scope because I see that as a refactoring: First, generalize the existing mechanism of tacking game specific data onto textures, then use that to handle the Quake 2 specific data as well.

Or we could just add more members and move all of this stuff to materials when we get to it. But the way it is now we have two mechanisms side by side that achieve the same ends.

Reply:
As a compromise, we can also leave it as is and just add a comment to clean this up when we introduce materials.

Reply:
Yeah, I added a TODO.